README.rst

```diff
@@ -62,12 +62,6 @@ Contributing
 See `our complete contributor's guide <CONTRIBUTING.md>`_ for more details.
 
 
-License
```

Comment:
I think it wouldn't hurt to keep this.

Reply:
I agree. Let just change the LICENSE file possibly. Can you provide this change?

Reply:
bump?